### DISCUSSION TITLE
ΓΕ1 Μέρος Δ παρουσίαση επιστημονικού άρθρου

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#18] [Q&A] [answered]
Καλημέρα σας.

Επειδή θεωρώ ότι η επικοινωνία μας θα γίνεται από εδώ κι όχι από το forum στο study.eap.gr, θα ήθελα να ρωτήσω τον κο Χωριανόπουλο πότε θα πρέπει κάποιος που ενδιαφέρεται για το Μέρος Δ (μπόνους) της εργασίας, να παρουσιάσει το θέμα της κι αν θα πρέπει να σας ειδοποιήσει από πριν.  Στις οδηγίες αναγράφεται ότι η παρουσίαση θα γίνει στην αμέσως επόμενη ΟΣΣ πριν την παράδοση της ΓΕ, άρα εννοείτε στις 29/10;

Ευχαριστώ,
Αγνή

**Comment by epidrome** [comment, marked as answer]
Ναι, η παρουσίαση του άρθρου θα γίνει στην (τυπική) ΟΣΣ1 για αυτό είναι προαιρετικό μπόνους, 

Ενθαρρύνουμε αυτήν την δραστηριότητα γιατί (εκτός από τον βαθμό) δείχνετε την τρέχουσα κατανόηση-ενδιαφέρονται που έχετε
